sentry/Chart.yaml

```diff
@@ -14,9 +14,10 @@ dependencies:
     version: 16.12.1
     condition: redis.enabled
   - name: kafka
-    repository: https://raw.githubusercontent.com/bitnami/charts/pre-2022/bitnami
-    version: 16.3.2
+    repository: oci://registry-1.docker.io/bitnamicharts
+    version: 22.1.3
     condition: kafka.enabled
+    alias: kafka2
```

Comment:
Do not change the dependency name. I noticed that in the Kafka chart from version 16 to version 21 "breaking" changes only in the zookeeper subchart in the auth section. Using an alias only complicates the update, since all kafka resources will be recreated

It is enough to indicate in the readme what you should pay attention to during upgrade, and what you need to do in order to stay on the zookeeper (`kafka.kraft.enabled` must be `false` and `kafka.zookeeper.enabled` must be `true`)

Reply:
I'll try the upgrade path to make sure it's possible.

Reply:
👍